src/pipecat/frames/frames.py

```diff
@@ -478,6 +478,12 @@ def __str__(self):
 class LLMMessagesFrame(DataFrame):
     """Frame containing LLM messages for chat completion.
 
+    .. deprecated:: 0.0.78
```

Comment:
Note to self: update this if for some reason this PR doesn't make it in before 0.0.78..